src/SignaturePad.InkPresenter.Shared/InkPresenter.cs

```diff
@@ -65,8 +72,13 @@ private bool ShouldRedrawBufferImage
 				var sizeChanged = false;
 				if (bitmapBuffer != null)
 				{
+#if NET471
+					var s = bitmapBuffer.Size;
```

Comment:
Rather add an extension method to the buffer type here: https://github.com/xamarin/SignaturePad/blob/master/src/SignaturePad.Shared/Extensions.cs

Then this code can have fewer `#if`

Reply:
I have reviewd all changes and remove redundant # everywhere